### PR TITLE
Add imodel datalocationid

### PR DESCRIPTION
**Pull request by azonys**

### CODE REVIEW

File: full-stack-tests/imodelhub-client/src/imodelhub/iModels.test.ts [hunk 1]
```diff
@@ -865,4 +868,10 @@ describe("iModelHub iModelsHandler", () => {
     mockUpdateiModel(projectId, imodel);
     await iModelClient.iModels.update(requestContext, projectId, imodel);
   });
+
+  it.only("should return DataLocationId", async () => {
```
Comment (pmconne) [inline full-stack-tests/imodelhub-client/src/imodelhub/iModels.test.ts:872]: Remove `.only`.

**Review by pmconne — APPROVED** [review]

File: full-stack-tests/imodelhub-client/src/imodelhub/iModels.test.ts (hunk 1 above)
Comment (azonys) [reply full-stack-tests/imodelhub-client/src/imodelhub/iModels.test.ts]: oh, removed.

**Review by Animagne — APPROVED** [review]

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**